src/platform/api.py

```diff
@@ -51,29 +74,78 @@ def _make_get_request(endpoint: str) -> dict:
         return data
 
 
+@beartype
+def delete_workstation(workstation: str):
+    """delete a workstation"""
+
+    return _make_request(
+        f"computebenches/{_get_org_id()}/{workstation}:delete", verb="POST"
+    )
+
+
+@beartype
+def delete_terminated_workstations():
+    """delete all terminated workstation"""
+
+    workstations = get_workstations()
+
+    names = [
+        ws["attributes"]["name"]
+        for ws in workstations
+        if ws["attributes"]["status"] == "TERMINATED"
+    ]
+
+    if len(names) == 0:
+        print("No workstations to delete")
+        return
+
+    org_id = _get_org_id()
+
+    with concurrent.futures.ThreadPoolExecutor() as executor:
```

Comment:
Can you bound the size of the thread pool? Or is it by default?

Reply:
the default is ncores